contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -8693,20 +8752,32 @@ handleClrUdtFuncCall(TSqlParser::Clr_udt_func_callContext *ctx)
 		for (size_t i = 0; i < method_calls.size(); ++i)
 		{
 			TSqlParser::Method_callContext *method = method_calls[i];
+			size_t ind = -1;
```

Comment:
can you point me to the test cases for these changes?

Reply:
Kindly check tests from line 642 of file `test/JDBC/expected/xml_exist-vu-verify.out`